spec/rails_helper.rb

```diff
@@ -5,63 +5,14 @@
 
 require 'spec_helper'
 require 'rspec/rails'
+require 'capybara/rails'
 require 'capybara/rspec'
-require 'database_cleaner'
+require 'selenium/webdriver'
 
 require 'support/factory_girl'
```

Comment:
This probably needs to be changed to `factory_bot`, right?

Reply:
I don't follow. why's that?

Reply:
@veganstraightedge this is blocked by https://github.com/spree/spree/issues/8450

I will open separate PR with this change to unblock this PR

Reply:
fixed here: https://github.com/crimethinc/website/pull/526

Reply:
Merged PR. Sorry for delay.